cmd/gitbase/command/server.go

```diff
@@ -107,6 +108,12 @@ func NewDatabaseEngine(
 func (c *Server) Execute(args []string) error {
 	if c.Verbose {
 		logrus.SetLevel(logrus.DebugLevel)
```

Comment:
so maybe we don't need verbose anymore. it kind of duplicates log-level.
so by default we'll have info, but if you need more set up to debug.

Reply:
It does but I also didn't want to break anything that used `-v`. Maybe we can deprecate it and hide the option from the help but continue supporting it.

Reply:
Of course, but right now -v and --log-level are exclusive. In other words, if you specify `-v` then `--log-level` is ignored. Maybe we keep the old `-v` but let --log-level to overwrite it (so move it outside `else`) - WDYT?

Reply:
I'll change it to do as you say.

Reply:
Changed the behavior and updated documentation.